app/script/user/AvailabilityMapper.js

```diff
@@ -37,6 +37,20 @@ z.user.AvailbilityMapper = (() => {
     }
   };
 
+  const protoFromType = availabilityType => {
+    switch (availabilityType) {
+      case z.user.AvailabilityType.AVAILABLE:
+        return z.proto.Availability.Type.AVAILABLE;
+      case z.user.AvailabilityType.AWAY:
+        return z.proto.Availability.Type.AWAY;
+      case z.user.AvailabilityType.BUSY:
+        return z.proto.Availability.Type.BUSY;
+      case z.user.AvailabilityType.NONE:
+        return z.proto.Availability.Type.NONE;
+      default:
```

Comment:
shouldn't be the `default` z.proto.Availability.Type.NONE.. so maybe replacing the line `48` with the `default`

Reply:
No, I want to be explicit. Added throwing errors for the default cases instead.